new/LIT7275QeneWaEsagged.xml

```diff
@@ -0,0 +1,72 @@
+                        <title xml:lang="en" corresp="#t1">Oh! I prostrate on my knees and in my heart I praise by the wood of the holy cross ...</title>
+                        <editor role="generalEditor" key="AB"/>
+                        <editor key="CH"/>
+                        <funder>Akademie der Wissenschaften in Hamburg</funder>
+                    </titleStmt>
+                    <publicationStmt>
+                        <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                        <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                        <pubPlace>Hamburg</pubPlace>
+                        <availability>
+                            <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                        </availability>
+                    </publicationStmt>
+                    <sourceDesc>
+                        <p/>
+                    </sourceDesc>
+                </fileDesc>
+                <encodingDesc>
+                    <p>A digital born TEI file</p>
+                </encodingDesc>
+                <profileDesc>
+                    <creation/>
+                    <abstract>
+                        <p>Neither type nor author of the qǝne are indicated in the manuscript. According to the classification by 
+                            <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl>,
+                            a qǝne of the wāzemā type.</p>
+                    </abstract>
+                    <textClass>
+                        <keywords>
+                            <term key="ChristianLiterature"/>
+                            <term key="Poetry"/>
+                            <term key="Qene"/>
+                            <term key="Wazema"/>
+                        </keywords>
+                    </textClass>
+                    <langUsage>
+                        <language ident="en">English</language>
+                        <language ident="gez">Gǝʿǝz</language>
+                    </langUsage>
+                </profileDesc>
+                <revisionDesc>
+                    <change who="CH" when="2025-02-11">Created entity</change>
+                </revisionDesc>
+            </teiHeader>
+            <text>
+                <body>
+                    <div type="edition" xml:lang="gez">
+                        <note>This edition is based on <ref type="mss" corresp="BNFabb145"/>, fol. 56rb. Division into verses is indicated with ። . 
+                            The end of the poem is indicated with ። ። .</note>
+                        <ab>
+                            <l n="1">ዋ። እሰግድ፡ በብረክየ፡ ወበልብየ፡ እገኒ፡ በዕፀ፡ ቅዱስ፡ መስቀል።</l> 
```

Comment:
I do not know what it may mean. May be, the text was corrupted here and smth is missing.

Reply:
I would guess that it's an indication of the qəne type: ዋ = ዋዜማ፡